deploy/goerli/.env.example

```diff
@@ -14,6 +14,8 @@ IPFS_PIN_ENDPOINTS=/dns/ipfs.infura.io/tcp/5001/https,/dns/ipfs/tcp/5001/http
 IPFS_PINATA_API_KEY=<pinata_api_key>
 IPFS_PINATA_SECRET_KEY=<pinata_secret_key>
 
+SCORING_DATABASE_PATH=/data/operator.db
```

Comment:
It should be network specific, otherwise it won't be possible to run oracle for multiple networks